plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/AbstractXmlValidator.java

```diff
@@ -67,10 +67,10 @@ static void deleteMarkers(IResource resource) throws CoreException {
   /**
    * Creates a marker from a given {@link BannedElement}
    */
-  static void createMarker(IResource resource, BannedElement element,
-      int elementOffset, String markerId, int severity) throws CoreException {
-    IMarker marker = resource.createMarker(markerId);
-    marker.setAttribute(IMarker.SEVERITY, severity);
+  static void createMarker(IResource resource, BannedElement element, int elementOffset)
```

Comment:
Do we need to pass elementOffset here? Is it different than 

element.getStart().getLineNumber()?